test/modules/aws-acm-list-integration.ts

```diff
@@ -24,10 +24,16 @@ describe('AwsAcmList Integration Testing', () => {
   it('installs the aws_account module', install(['aws_account']));
 
   it('inserts aws credentials', query(`
-    INSERT INTO aws_account (region, access_key_id, secret_access_key)
-    VALUES ('${process.env.AWS_REGION}', '${process.env.AWS_ACCESS_KEY_ID}', '${process.env.AWS_SECRET_ACCESS_KEY}')
+    INSERT INTO aws_credentials (access_key_id, secret_access_key)
+    VALUES ('${process.env.AWS_ACCESS_KEY_ID}', '${process.env.AWS_SECRET_ACCESS_KEY}')
   `, undefined, false));
 
+  it('syncs the regions', sync());
+
+  it('sets the default region', query(`
+    UPDATE aws_regions SET is_default = TRUE WHERE region = '${process.env.AWS_REGION}';
```

Comment:
does this work if thers already a `is_defualt = TRUE`? I think at this point we will have 2 regions as default in the regions table maybe this query will need to be something like:
```sql
update aws_regions
set is_default = case is_default 
    when FALSE then TRUE
    else FALSE
    end
where region = '${process.env.AWS_REGION}' OR is_default = TRUE;
```

Have not tested it but something like that should work.

Reply:
I see your point. I was thinking further by the time you add the constraint to always having only one `is_default`, by the time you `sync` you will need to fix that is_default to a fallback value, and then I guess you will have to do something like what I've suggested maybe.

I know this is WIP, I'm just thinking out loud.

Reply:
Oh, on that point I plan to have a `default_aws_region` postgres function in this PR. If called with no arguments it returns the current default region (or it automatically sets `us-east-1` as the default region for you and returns that), while if you call it with a string argument, it changes the default region to the one you provided and then returns that region back to you (or errors if the string provided is not a region).

Reply:
once we generalize cloud APIs (https://github.com/iasql/iasql-engine/issues/1152) it might make sense to provide credentials on the very first function call to `install_provider('aws', '*****', '******')` such that regions are filled in from the start?